iothub/device/src/GlobalSuppressions.cs

```diff
@@ -11,3 +11,4 @@
 [assembly: SuppressMessage("Design", "CA1031:Do not catch general exception types", Justification = "SDK hides non-actionable errors from user", Scope = "module")]
 [assembly: SuppressMessage("Naming", "CA1707:Identifiers should not contain underscores", Justification = "Public API cannot be changed.", Scope = "type", Target = "~T:Microsoft.Azure.Devices.Client.ConnectionStatusChangeReason")]
 [assembly: SuppressMessage("Naming", "CA1707:Identifiers should not contain underscores", Justification = "Public API cannot be changed.", Scope = "type", Target = "~T:Microsoft.Azure.Devices.Client.ConnectionStatus")]
+[assembly: SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification = "The authentication method disposed when the client is disposed.", Scope = "member", Target = "~M:Microsoft.Azure.Devices.Client.Edge.EdgeModuleClientFactory.CreateInternalClientFromEnvironmentAsync~System.Threading.Tasks.Task{Microsoft.Azure.Devices.Client.ModuleClient}")]
```

Comment:
I don't understand the phrase "the authentication method disposed". How is a method disposed?

Reply:
I've updated the text to say: IDisposable ModuleAuthenticationWithHsm is disposed when the client is disposed. (the "authentication method" here was referencing the IAuthenticationMethod type)